components/match2/wikis/arenafps/legacy/match_maps_legacy.lua

```diff
@@ -0,0 +1,196 @@
+---
+-- @Liquipedia
+-- wiki=arenafps
+-- page=Module:MatchMaps/Legacy
```

Comment:
since this module does 3 jobs
- matchlists conversion
- bms conversion
- showmatch conversion

i am not sure if the name is fitting
the old MatchMaps (afaik) were only part of matchlists

Reply:
I can split BracketMatchSummary stuff, but I haven't a name suggestion for MatchMaps module :D.